pom.xml

```diff
@@ -44,31 +44,17 @@
                 <directory>src/main/resources</directory>
                 <filtering>true</filtering>
                 <includes>
-                    <include>paper-plugin.yml</include>
+                    <include>plugin.yml</include>
```

Comment:
This plugin only works on Paper and derivatives, because it makes use of the PaperServerListPingEvent, which does not exist in Spigot. Having only the paper-plugin.yml instead of the plugin.yml prevents this plugin from even loading on Spigot servers.

Reply:
The paper-plugin.yml only works with paper 1.20+ plugin.yml works with PAPER 1.17+ with that config. Also Spigot works too that my fault, but it was the only way I could think of. You could use this or not. I will use the fork it is working on Paper 1.17 to 1.20.4

Reply:
If you don't want to use it in Spigot, close the pull request

Reply:
> If you don't want to use it in Spigot
It simply doesn't work in Spigot, as Spigot doesn't provide the necessary event.

Reply:
But with this changes will work in Paper 1.17+ if not no...